accord-core/src/main/java/accord/local/Command.java

```diff
@@ -666,6 +673,11 @@ public final boolean isTruncated()
         return status().hasBeen(Status.Truncated);
     }
 
+    public static boolean isTruncated(Status status)
+    {
+        return status.hasBeen(Status.Truncated);
```

Comment:
I agree with David that this could be misleading, as it can include `Invalidated` and `is` should mean `=` not `>=`.

Reply:
Removed this part. Turns out it was not necessary after refactorings.